### ISSUE TITLE
Control FPS

### ISSUE THREAD

**Issue by xcracing** [kimmobrunfeldt/progressbar.js#189]
Is there a way to control the FPS rendered?   I have an application where I'm creating numerous progress bars that all have a duration of approx 120 sec, all on a low power CPU device.   I would like to lower the FPS in order to recover some CPU time for other functions.

**Comment by kimmobrunfeldt** [comment]
Interesting question. I'm using https://github.com/jeremyckahn/shifty for tweening values. If you can figure out a good way to control the fps, I'm open to hear how it could be implemented. If it sounds good, I'm open for a PR.

**Comment by xcracing** [comment]
Thanks for the reply.   I am by no means a JS expert, I tinkered with things a little bit to see if I could alter the FPS, but nothing I did seemed to have an effect - also queried google quite a bit to no avail.   I'll do some more digging as time allows.

The below tweening library has control for FPS - I'll have to dig around a little and see if it might be feasible to use it instead.

https://github.com/JakeSidSmith/slik
